deprecations/README.md

```diff
@@ -39,6 +39,12 @@ The table columns have the following meanings:
 
 | Feature | Comments | Deprecated | Discontinued |
 | :-- | :-- | :-- | :-- |
+| `TaxIdentifier` in [Close bill](../operations/bills.md#close-bill) | Replaced by `AccountTaxIdentifier` | 16 Nov 2023 | 16 Nov 2024 |
```

Comment:
The two entries here for `TaxIdentifier` appear in contradiction, I think it should be one entry. On the surface, it seems like `TaxIdentifier` is replaced by `AccountTaxIdentifier` which is identical except for the name, the same as for `Address` below. So why is it deprecated?

Reply:
The second one should have been 'CompanyTaxIdentifier'. changed it